.github/workflows/prerelease.yml

```diff
@@ -341,15 +337,16 @@ jobs:
           apt-get update
           apt-get -y install git fakeroot build-essential \
             ncurses-dev xz-utils libssl-dev bc flex libelf-dev bison \
-            bear \
             lld-${{ matrix.llvm-version }}
 
-      - name: Clone and build the Linux kernel to create a compilation database
+      - name: Clone, configure, and build the Linux kernel
         run: |
-          git clone --depth=1 https://github.com/torvalds/linux.git linux
+          git clone https://github.com/torvalds/linux.git --depth=1 --branch=v6.10 linux
           cd linux
-          bear -- make defconfig
-          bear -- make LLVM=-${{ matrix.llvm-version }} -j $(nproc)
+          make defconfig LLVM=-${{ matrix.llvm-version }}
+          make prepare LLVM=-${{ matrix.llvm-version }}
+          make LLVM=-${{ matrix.llvm-version }} -j $(nproc)
+          python3 ./scripts/clang-tools/gen_compile_commands.py
           cd ../
```

Comment:
Sorry, I missed this before. I believe you do not need this `cd ../` command, as the next step again should start in the "original" working directory (or in a directory set by the `working-directory` option)